### PR TITLE
fix README usage markdown

### PR DESCRIPTION
**Pull request by kerneljake**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ A utility script (./tpccbenchmark) is provided for running the benchmark. The op
    --runscript &lt;arg&gt;         Run an SQL script
 -s,--sample &lt;arg&gt;            Sampling window
 -v,--verbose                 Display Messages
-```
+</pre>
```
Comment (robertsami) [inline README.md:63]: @kerneljake is this change necessary? what does it do functionally? my understanding is that the previous was valid markdown as well?

Comment (kerneljake) [reply README.md:63]: The previous markdown does not render the usage message as proper HTML in a browser, plus the text columns are not aligned.  Backticks are syntactically correct markdown, but they yield literal `&lt;` when viewed in a browser, whereas &lt;pre&gt; renders angled brackets correctly.

Comment (kerneljake) [reply README.md:63]: Hah! github markdown clobbered my previous comment.  I was trying to illustrate literal "ampersand ell tee semicolon" versus angled bracket.

**Comment by polarweasel** [comment]
(Closed because this was the wrong approach. Fixed now.)
